pom.xml

```diff
@@ -137,41 +136,68 @@
             <url>http://repo.jenkins-ci.org/public/</url>
         </pluginRepository>
     </pluginRepositories>
-  
-  <build>
-	<plugins>
-		<plugin>
-		<groupId>org.jenkins-ci.tools</groupId>
-		<artifactId>maven-hpi-plugin</artifactId>
-		<extensions>true</extensions>
-		<configuration>
-				<compatibleSinceVersion />
-		</configuration>
-		</plugin>
-		<plugin>
-			<groupId>org.apache.maven.plugins</groupId>
-            <artifactId>maven-release-plugin</artifactId>
-            <version>2.5</version>
-         </plugin>
-         <plugin>
-             <groupId>org.apache.maven.plugins</groupId>
-             <artifactId>maven-surefire-plugin</artifactId>
-             <configuration>
-                 <excludes>
-                     <exclude>InjectedTest.java</exclude>
-                 </excludes>
-             </configuration>
-         </plugin>
-         <plugin>
-             <groupId>org.apache.maven.plugins</groupId>
-             <artifactId>maven-failsafe-plugin</artifactId>
-             <configuration>
-                 <forkCount>10</forkCount>
-                 <reuseForks>true</reuseForks>
-                 <threadCountMethods>10</threadCountMethods>
-                 <parallel>methods</parallel>
-             </configuration>
-         </plugin>
-	</plugins>
-  </build>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.jenkins-ci.tools</groupId>
+                <artifactId>maven-hpi-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <compatibleSinceVersion/>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-release-plugin</artifactId>
+                <version>2.5</version>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <excludes>
+                        <exclude>InjectedTest.java</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <configuration>
+                    <forkCount>10</forkCount>
+                    <reuseForks>true</reuseForks>
+                    <threadCountMethods>10</threadCountMethods>
+                    <parallel>methods</parallel>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
`maven-checkstyle-plugin` configuration, which does the following

* Inject a `checkstyle:checkstyle` action into the `validate` phase. It will fail the build if the style check is not passed.
* specify the checkstyle configuration file with the `configLocation` property.
* exclude the checks for the generated message source.